orders/views.py

```diff
@@ -1,3 +1,78 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.views         import View
+from django.http          import JsonResponse
+from django.db            import transaction, IntegrityError
+from django.db.models     import Sum, F
+
+from orders.models        import Order, OrderItem, OrderStatus, OrderItemStatus, PaymentMethod
+from users.models         import Cart
+from users.utils          import login_decorator
+
+class OrderView(View):
+    @login_decorator
+    def post(self,request):
+        with transaction.atomic():
+            data           = json.loads(request.body)
+            user           = request.user
+            carts          = Cart.objects.filter(user_id = user.id).select_related("user","product")
+            payment_method = data.get('payment')
+            
+            order = Order.objects.create(
+                user            = user,
+                status          = OrderStatus.objects.get(status='입금확인중'),
+                payment_method  = PaymentMethod.objects.get(name = payment_method)
+            )
+            
+            OrderItem.objects.bulk_create([OrderItem(
+                    product  = cart.product,
+                    order    = order,
+                    status   = OrderItemStatus.objects.get(status='주문완료'),
```

Comment:
정규화된 데이터의 값을 사용할 때는 Enum Class로 관리하는 것이 좋습니다. (하드 코딩하지 않고 사용할 수 있습니다.)
Django 공식문서에서 Enum 키워드로 검색해보시고 적용해보세요~

```suggestion
                    status   = OrderItemStatus.objects.get(status='주문완료'),
```

Reply:
확인하고 반영하겠습니다